### ISSUE TITLE
Syntax Highlighting for Updated Code Lines

### ISSUE THREAD

**Issue by genefyx** [microsoft/vscode-copilot-release#498]
I'd like to suggest a feature for Copilot: Can it color code lines that have been updated or changed? This would make it much easier for me to quickly spot the modifications without having to scan through the entire code.

**Comment by jrieken** [comment]
@genefyx Uou mean inline chat, right? I am confused because it does show a highlight/background color and also shows an embedded diff editor for the changes

**Comment by genefyx** [comment]
So the extension for copilot chat, when making suggestions for changes.
It is showing me a function for example with no indication of what lines or portions of the line
that have changed, so I have to diff myself, or review it myself.

I ran it again just in case, and there is no background highlight or color change

**Comment by jrieken** [comment]
please attach a gif

**Comment by genefyx** [comment]
![code_screenshot](https://github.com/microsoft/vscode-copilot-release/assets/30269995/35bd3c5e-e965-4570-9b56-ac0c25034cd7)

**Comment by genefyx** [comment]
Yes, (CMD+I) appears to work, it appears it be a trade off between clearer implementation of changes vs clearer description. Thank you. Can this be considered for chat in the future or is it working as intended?
